woocommerce/woocommerce-android
# Analytics Performance: adjust analytics hub to use stored data

Summary
==========
Partially fix issue #9269 by connecting the newly introduced `AnalyticsUpdateDataStore` with the `UpdateAnalyticsHubStats` use case, effectively connecting the expected logic for updating the stats inside the Analytics Hub.

How to Test
==========
1. Open the app and enter the Analytics Hub
2. Once the data is loaded, exit and reenter the Hub with the same range selection, and verify that no network operation is triggered.
3. Wait 30 seconds, exit the Hub, and enter again. Now verify that a new request was made fetching the stats data inside the Hub again for that specific selection range.

Update release notes:
